tests/api/api/utils.py

```diff
@@ -54,10 +54,13 @@ async def create_project_async(
     return resp
 
 
-def _get_project_obj(project_name) -> mlrun.api.schemas.Project:
+def _get_project_obj(project_name, artifact_path) -> mlrun.api.schemas.Project:
```

Comment:
```suggestion
def _create_project_obj(project_name, artifact_path) -> mlrun.api.schemas.Project:
```
Seems a better name, given that it creates a new project object.